cypress/integration/index.spec.js

```diff
@@ -0,0 +1,38 @@
+/* global describe, it, cy */
+// Source: https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/By_example/Detect_WebGL
+const detectWebGLContext = () => {
+  /* eslint-disable no-console */
+  const canvas = document.createElement('canvas');
+  const gl =
+    canvas.getContext('webgl') || canvas.getContext('experimental-webgl');
+  if (gl && gl instanceof WebGLRenderingContext) {
+    console.info('Congratulations! Your browser supports WebGL.');
+  } else {
+    console.info(
+      'Failed to get WebGL context. Your browser or device may not support WebGL.'
+    );
+  }
+};
+
+detectWebGLContext();
```

Comment:
Why would an integration test care about the backend that's being used?
Or are you thinking this will be used later on for benchmarking (and that's why we want to know which backend we're running on)

Reply:
That was initially to troubleshoot the WebGL support on the Electron/Chrome browsers used by Cypress.IO (see my comment above).
But now that you mention it, it could be useful for benchmark tests.